docs/imagebackground.md

```diff
@@ -9,14 +9,27 @@ You might not want to use `<ImageBackground>` in some cases, since the implement
 
 Note that you must specify some width and height style attributes.
 
-## Example
-
 ```jsx
-return (
   <ImageBackground source={...} style={{width: '100%', height: '100%'}}>
     <Text>Inside</Text>
   </ImageBackground>
-);
+```
+
+## Example
+
+```SnackPlayer name=ImageBackground
+  import React from 'react';
+  import {View, ImageBackground, Text} from 'react-native'
+
+  export default function App() {
+    return (
+      <View>
+        <ImageBackground source={{uri: 'https://facebook.github.io/react/logo-og.png'}} style={{width: '100%', height: '100%'}}>
```

Comment:
I would place `source` and `style` in separate lines (like for example [https://facebook.github.io/react-native/docs/next/touchablenativefeedback](TouchableNativeFeedback)) since Snack code editor is quite narrow.